Library/Formula/rolldice.rb

```diff
@@ -12,7 +12,7 @@ def install
 
   test do
     output = `#{bin}/rolldice -s 1x2d6`
-    assert output.include?("Roll #1")
+    assert_equal true, output.include?("Roll #1")
```

Comment:
It’d be awesome if you could replace the whole test with `assert_match "Roll #1", shell_output("#{bin}/rolldice -s 1x2d6")`